src/gcp_scanner/crawl.py

```diff
@@ -95,8 +95,8 @@ def get_project_list(credentials: Credentials) -> List[Dict[str, Any]]:
     while request is not None:
       response = request.execute()
 
-      for project in response.get("projects", []):
-        project_list.append(project)
+      project_list = [project 
+                      for project in response.get("projects",[])]
```

Comment:
Redundant.

Reply:
I have made the requested changes in the latest PR. Check and let me know if any changes required.